internal/users/edit.go

```diff
@@ -14,34 +14,19 @@ import (
 	"golang.org/x/crypto/bcrypt"
 )
 
-type EditUserRequest struct {
-	ID        int    `json:"-"`
-	Name      string `json:"name" validate:"omitempty,checkName"`
-	Password  string `json:"password" validate:"omitempty,checkPassword"`
-	Email     string `json:"email" validate:"omitempty,email"`
-	BirthDate string `json:"birthdate" validate:"omitempty,checkDate"`
-	Nickname  string `json:"nickname" validate:"omitempty,checkNickname"`
-	Bio       string `json:"bio" validate:"omitempty,checkBio"`
-	Location  string `json:"location" validate:"omitempty,checkLocation"`
+type editUserRequest struct {
+	createUserRequest
 }
```

Comment:
```suggestion
type editUserRequest createUserRequest
```
can do alias, would be better in this case

Reply:
actually no, it's a different request with different validation rules